### ISSUE TITLE
Radio button navigation out of sync with focus

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#107]
In a recent test, @terracoda discovered that the navigation of radio buttons is out of sync with the focus highlight.  The visual focus can be off by one keystroke, and arrow keys do not focus the input element correctly.

I have not been able to reproduce this.  Here is a quick screen cast of my navigation with the radio buttons:
![balloons](https://cloud.githubusercontent.com/assets/6396244/11984426/5bb73630-a978-11e5-9804-3d3a0c7421a9.gif)

Using the arrow keys highlights and selects the appropriate input element.  This behavior may be incorrect, or it might be broken for some screen readers.

**Comment by jessegreenberg** [comment]
Additional comments from @terracoda:

> Jaws keeps focus and activation as two separate steps. I think the Jaws approach makes more sense from a non-visual perspective. I listen to the label and checked state and then I decide if I want to change it. There is some debate in the a11y community on this topic. If I am not looking at the pink focus and just listening, I understand the interaction.

**Comment by jessegreenberg** [comment]
I am wondering if https://github.com/phetsims/sun/commit/bf633233bf0f68771604c7f77829045153d8d2a7 also fixed this issue now that focus should first snap to the checked radio button?

**Comment by jessegreenberg** [comment]
This behavior is specific to the accessibility API, and it is up to screen readers to handle this focus.  I observed this issue in JAWS.  For JAWS, the radio button does not receive focus until selected by the user.  This is the behavior for native HTML radio groups as well.

Upon initial focus of the radio group, focus snaps to the currently selected radio button.  I am afraid that is all we can do here for now.  Closing.
